controllers/tokenController.js

```diff
@@ -0,0 +1,40 @@
+const User = require("../models/users");
+const sequelize = require("../database/db");
+let crypto = require("crypto");
+const Access_Token = require("../models/access_token");
+
+module.exports = {
+  createToken: async (req, res) => {
+    try {
+      const { username } = req.body;
+      const found = await User.findOne({
+        where: {
+          username,
+        },
+      });
+      if (!found) {
+        return res.status(400).json({ message: "Invalid credentials" });
+      }
+
+      const user_id = found.id;
```

Comment:
use es6 method to create user_id and username variable

Reply:
in es6 const is used to create variables, and the same i used.

Reply:
read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment and implement

Reply:
has been resolved.